mac

```diff
@@ -150,6 +150,11 @@ brew_install_or_upgrade 'node'
 brew_install_or_upgrade 'rbenv'
 brew_install_or_upgrade 'ruby-build'
 
+brew_install_or_upgrade 'caskroom/cask/virtualbox'
```

Comment:
I'm not sure if we can directly install from brew cask like this, or if we need to go the longer route:

```bash
brew tap caskroom/cask
brew install brew-cask
brew cask install  virtualbox
```